var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Timemory(CMakePackage):
+    """Timing + Memory + Hardware Counter Utilities for C/C++/CUDA/Python"""
+
+    homepage = 'https://timemory.readthedocs.io/en/latest/'
+    git = 'https://github.com/NERSC/timemory.git'
+    maintainers = ['jrmadsen']
+
+    version('master', branch='master', submodules=True)
```

Comment:
I asked about stable release rather than master version (which is pain to re-deploy). I asked about the new release [here](https://github.com/NERSC/timemory/issues/27).

Reply:
The next version won't happen before a couple of weeks. What about merging this now in BlueBrain/spack and then promote the patch upstream in spack/spack when timemory releases a new stable version?

Reply:
@tristan0x : As master is moving target, what if we do [similar to steps](https://github.com/BlueBrain/spack/pull/530)? i.e. tag  alpha version ourselves from current master?
```suggestion
    version('develop', branch='master', submodules=True)
    version('3.0.0a', commit='36520ff4aedc09ea7b54981dd8c97c4960f6ee79', submodules=True)
```